src/Libraries/RevitNodes/Elements/Element.cs

```diff
@@ -197,23 +197,19 @@ public Category GetCategory
         }
 
         /// <summary>
-        /// Returns the FamilyType for this Element. Returns null if the Element cannot have a FamilyType assigned.
+        /// Returns the ElementType for this Element. Returns null if the Element cannot have an ElementType assigned.
         /// </summary>
         /// <returns name="ElementType">Element Type or Null.</returns>
-        public Element ElementType
+        public ElementType ElementType
```

Comment:
@radumg here is an example of a simple API break - this class no longer has as property called `Element`.